config/initializers/traffic_control.rb

```diff
@@ -0,0 +1,2 @@
+ActiveJob::TrafficControl.client = Redis.new(Rails.application.config.cache_store[1])
```

Comment:
Please add a comment that this initializer depends on the cache store initializer being run first and the cache store being redis.